src/mscorlib/GenerateCompilerResponseFile.targets

```diff
@@ -16,7 +16,8 @@
       <Clean Include="$(IntermediateOutputPath)\moduleName.rsp" />
     </ItemGroup>
     <PropertyGroup>
-      <CompilerResponseFile>$(IntermediateOutputPath)\moduleName.rsp</CompilerResponseFile>
+      <CompilerResponseFile Condition="'$(CompilerResponseFile)'!=''">$(IntermediateOutputPath)\moduleName.rsp;$(CompilerResponseFile)</CompilerResponseFile>
```

Comment:
I'll test to make sure that we can make that a one-liner as you suggest.

I tested with building the root build.proj project, and verified that all 4 projects(which includes System.Private.CoreLib) that actually invoke csc.exe are passing the checksum flag now.